src/index.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+import './index.css';
+import App from './App';
+import reportWebVitals from './reportWebVitals';
+
+const root = ReactDOM.createRoot(document.getElementById('root'));
+root.render(
+  <React.StrictMode>
+    <App />
+  </React.StrictMode>
```

Comment:
This also is optional to work on it instead of importing react, you can... import {StrictMode} from 'react';
 ```suggestion
  <StrictMode>
    <App />
  </StrictMode>
```